### ISSUE TITLE
Make clear contributions are contributions in plots

### ISSUE THREAD

**Issue by raoulcollenteur** [pastas/pastas#697]
**Describe the proposed enhancement**

Too often users do not understand the contribution is the contribution, and not, for example, the recharge. I guess we might partly solve this through better ylabels or legends..

**Comment by mbakker7** [comment]
But how? A y-label that says "contribution of recharge to head" is a bit long. 
The graph I always miss is one that shows the head observations and model, and then graphs below it for each of the stresses.

**Comment by raoulcollenteur** [comment]
What about simply using the ylabel "rise [L]" (or with adapted units) ? Short and simple. I think Menyanthes did that right?

**Comment by rubencalje** [comment]
> What about simply using the ylabel "rise [L]" (or with adapted units) ? Short and simple. I think Menyanthes did that right?

I agree. I am in favor of just setting the units on each of the y-axes in `ml.plots.results` and other methods like `ml.plots.decomposition`. We should then make `unit` a property of the Model-class, with a default value of "m". I would not put the word "rise" or "change" on the y-axis, as this increases the required space, will give room for discussion, and is language-dependent.

We can later also think of a method to implement a unit for each of the stresses, which we can also show, for example in `ml.plots.series()`.

**Comment by mbakker7** [comment]
There are two issues at stake here: Do we put something on the vertical axis and are we going to provide units. I
I think we should put something on the vertical axis. The word 'rise' is about the shortest I can think off. And is descriptive. Except for the top graph, which shows heads. 

I am a bit reluctantly to start keeping track of units everywhere. I am against a default. So if nothing is entered, nothing is printed. And the users can already put something along the axes themselves (if they know a bit of matpotlib). 

In conclusion, +1 for 'rise', -1 for giving units.

**Comment by martinvonk** [comment]
Can't we do something visually? Instead of y-labels. 

An addition sign below the oseries / simulation Axes:

![image](https://github.com/pastas/pastas/assets/66305055/22ccbffa-d530-4b1f-a06e-950d537b91fa)

Or give Axes that contribute to the simulated head a slightly different background color:

![image](https://github.com/pastas/pastas/assets/66305055/67cbdba9-3e00-4971-85cb-c9d2f0074070)

**Comment by raoulcollenteur** [comment]
This would be my proposal:

![Unknown](https://github.com/pastas/pastas/assets/12051167/06a1282e-4c6a-47be-ba37-a49db4a71d67)

**Comment by raoulcollenteur** [comment]
Closing issue after merging PR #723 . If someone wants to add optional Units, a new separate issue can be started.
